examples/postfilter1.py

```diff
@@ -0,0 +1,42 @@
+########################################################################
+#
+#       Author:  The Blosc development team - blosc@blosc.org
+#
+########################################################################
```

Comment:
For the header, better stick with something standard like:

```
#######################################################################
# Copyright (C) 2019-present, Blosc Development team <blosc@blosc.org>
# All rights reserved.
#
# This source code is licensed under a BSD-style license (found in the
# LICENSE file in the root directory of this source tree)
#######################################################################
```

Reply:
Fixed in 8fe520bc3b8fdaa36991a8037bec4163a263596e.